docs/resources/SmbCommonConfig.md

```diff
@@ -0,0 +1,33 @@
+---
+title: SmbCommonConfig Custom Resource
+---
```

Comment:
Is this standard markdown syntax for title? I was not aware of it. Anyhow, I think the rendering on Github is not what you wanted to have. Better change to ```# SmbCommonConfig Custom Resource```

Reply:
I borrowed it from the rook docs. See https://github.com/rook/rook/blob/master/Documentation/Storage-Configuration/NFS/nfs.md

I guess it's not really meant for "github" view rendering. But since we don't render our docs like rook maybe it is overkill. I'll think about it.

Reply:
> Is this standard markdown syntax for title? I was not aware of it. Anyhow, I think the rendering on Github is not what you wanted to have. Better change to `# SmbCommonConfig Custom Resource`

+1

Reply:
done